makefiles/features_modules.inc.mk

```diff
@@ -11,9 +11,12 @@ USEMODULE += $(PERIPH_FEATURES)
 
 # Add all USED periph_% init modules unless they are blacklisted
 PERIPH_IGNORE_MODULES := \
+	periph_cipher_aes_128_cbc \
   periph_clic \
   periph_common \
   periph_coretimer \
+  periph_ecc_p192r1 \
+	periph_ecc_p256r1 \
```

Comment:
oops

Reply:
What exactly does "oops" mean?

Reply:
The mismatched indentation I would assume